starknet_devnet/predeployed_contract_wrapper.py

```diff
@@ -1,9 +1,100 @@
 """Parent for predeployed contract wrapper classes"""
+from abc import ABC, abstractmethod
 
+from starkware.starknet.testing.starknet import Starknet
+from starkware.starknet.services.api.contract_class import ContractClass
+from starkware.starknet.testing.contract import StarknetContract
 
-class PredeployedContractWrapper:
+
+class PredeployedContractWrapper(ABC):
     """Parent class for all predeployed contract wrapper classes"""
 
+    # Cannot import it because of circular imports
```

Comment:
Regarding this, you can see how it has been handled so far in e.g. account.py

Reply:
All imports doesn't work here because of:

```
ImportError while loading conftest '/Users/mac/Workspace/OpenSource/starknet-devnet/test/conftest.py'.
test/conftest.py:10: in <module>
    from .util import run_devnet_in_background, terminate_and_wait
test/util.py:21: in <module>
    from starknet_devnet.general_config import DEFAULT_GENERAL_CONFIG
starknet_devnet/general_config.py:15: in <module>
    from .fee_token import FeeToken
starknet_devnet/fee_token.py:14: in <module>
    from starknet_devnet.chargeable_account import ChargeableAccount
starknet_devnet/chargeable_account.py:5: in <module>
    from starknet_devnet.account import Account
starknet_devnet/account.py:14: in <module>
    from starknet_devnet.predeployed_contract_wrapper import PredeployedContractWrapper
starknet_devnet/predeployed_contract_wrapper.py:9: in <module>
    class PredeployedContractWrapper(ABC):
starknet_devnet/predeployed_contract_wrapper.py:13: in PredeployedContractWrapper
    from .starknet_wrapper import StarknetWrapper
starknet_devnet/starknet_wrapper.py:60: in <module>
    from .accounts import Accounts
starknet_devnet/accounts.py:11: in <module>
    from .account import Account
E   ImportError: cannot import name 'Account' from partially initialized module 'starknet_devnet.account' (most likely due to a circular import) (/Users/mac/Workspace/OpenSource/starknet-devnet/starknet_devnet/account.py)
```